src/commands/batch.py

```diff
@@ -39,14 +39,15 @@ def history(ctx, **options):
                          .filter(Batch.id == int(batch_id))
         jobs = query.all()
         jobs = sorted(jobs, key=lambda job: job.created_date, reverse=True)
-        headers = ['Batch', 'Node', 'Command', 'Exit Code', 'Date']
+        headers = ['Batch', 'Node', 'Command', 'Arguments', 'Exit Code', 'Date']
```

Comment:
I am not sure about including `arguments` in this table. They will be exactly the same for each `node` in a `batch`. We should save space in this table as it is already wide.

Reply:
Also move the `Exit Code` to be the third column of the table. This is discussed in a latter comment.